_tour/implicit-parameters.md

```diff
@@ -13,52 +13,58 @@ previous-page: self-types
 redirect_from: "/tutorials/tour/implicit-parameters.html"
 ---
 
-A method with _implicit parameters_ can be applied to arguments just like a normal method. In this case the implicit label has no effect. However, if such a method misses arguments for its implicit parameters, such arguments will be automatically provided.
+A method can have an _implicit_ parameter list, marked by the _implicit_ keyword at the start of the parameter list. If the parameters in that parameter list are not passed as usual, Scala will look if it can get an implicit value of the correct type, and if it can, pass it automatically.
 
-The actual arguments that are eligible to be passed to an implicit parameter fall into two categories:
+The places Scala will look for these parameters fall into two categories:
 
-* First, eligible are all identifiers x that can be accessed at the point of the method call without a prefix and that denote an implicit definition or an implicit parameter.
-* Second, eligible are also all members of companion modules of the implicit parameter's type that are labeled implicit.
+* Scala will first look for implicit definitions and implicit parameters that can be accessed directly (without a prefix) at the point the method with the implicit parameter block is called.
+* Then, it falls back to all members that are marked implicit in the companion object of the implicit parameter.
```

Comment:
What about "Then it looks for members marked implicit in all the companion objects associated with the implicit candidate type"? A link to https://docs.scala-lang.org/tutorials/FAQ/finding-implicits.html would be useful for those that want to know more.

Reply:
Linking to the implicit lookup FAQ there is a great idea.

The phrasing "in all the companion objects associated with the implicit candidate type" isn't as "human readable" as I'd like it to be, especially for the tour. It still reads a bit as legalese. I'll think on it some more. Maybe not saying much about where it looks exactly (but link to the FAQ for details) is another option.